ingest/src/sources/mongodb-university/MongoDbUniversityDataApiClient.ts

```diff
@@ -0,0 +1,179 @@
+import fetch from "node-fetch";
+/**
+  Item in the Thought Industries catalog as returned from the
+  MongoDB University Data API.
+ */
+export interface TiCatalogItem {
+  id: string;
+  ti_id: string;
+  /**
+        IDs for videos associated with the catalog item.
+       */
+  associated_videos: string[];
+  /**
+        Learning format for the catalog item.
+        @example "Unit" || "Course" || "Lab" || "...etc"
+       */
+  learning_format: string;
```

Comment:
Non-blocker: do we have a full list of this? we could switch to a discriminated union type, e.g. `"Unit" | "Course" | "Lab"` instead of just `string`.

Reply:
i don't offhand

Reply:
Certification', Course', 'Educator Content', 'Instructor Led', 'Lab', 'Learning Bytes', 'Learning Path', 'Resource', 'Unit',  'Video'